mods/ascent/_ascent.dme

```diff
@@ -31,4 +31,11 @@
 #include "structures\ship_furniture.dm"
 #include "turfs\ship.dm"
 // END_INCLUDE
-#endif
\ No newline at end of file
+#endif
+// BEGIN_INTERNALS
```

Comment:
some garbig here

Reply:
You need to manually remove it after opening the DME in DM